doc/modules/decomposition.rst

```diff
@@ -799,6 +799,10 @@ Maximizing ELBO is equivalent to minimizing the Kullback-Leibler(KL) divergence
 between :math:`q(z,\theta,\beta)` and the true posterior
 :math:`p(z, \theta, \beta |w, \alpha, \eta)`.
 
+The :attr:`n_topics` of :class:`LatentDirichletAllocation` can be used to set the number of topics, :math:`K`.
+The :attr:`doc_topic_prior` can be used to set Dirichlet Prior, :math:`\alpha`, of the Document-Topic distribution, :math:`\theta`.
```

Comment:
Please break lines at 79 characters. The idea was to give a bit more background and explain what that actually means.